pyproject.toml

```diff
@@ -111,6 +111,8 @@ flatc = "executorch.data.bin:flatc"
   "*.fbs",
   # Some kernel libraries need their .yaml files.
   "*.yaml",
+  # Add trained models from backends/nxp/experimental
+  "backends/nxp/experimental/*.pth",
```

Comment:
Wrong path: Pretrained model is on the model is on `examples/nxp/experimental/cifar_net/cifar_net.pth`